app/packages/looker/src/elements/video.ts

```diff
@@ -645,7 +658,7 @@ export class VideoElement extends BaseElement<VideoState, HTMLVideoElement> {
 
     this.removeEvents();
     this.element = null;
-    this.release && this.release();
+    this?.release();
```

Comment:
Looks like a typo, `this` will always exist. I think this should be `this.release?.()` or the more readable `if (this.release) this.release()`